declare function onErrorResumeNext<T extends () => Promise<U>, U = unknown>(fn: T, context?: undefined): Promise<U | undefined>;
declare function onErrorResumeNext<T extends (this: V) => Promise<U>, U = unknown, V = undefined>(fn: T, context: V): Promise<U | undefined>;

export { onErrorResumeNext };
